site/OwidSocials.tsx

```diff
@@ -0,0 +1,33 @@
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
+import { RSS_FEEDS, SOCIALS } from "./SiteConstants.js"
+
+export const OwidSocials = ({
+    includeRss = false,
+}: { includeRss?: boolean } = {}) => {
+    const items = includeRss ? [...SOCIALS, ...RSS_FEEDS] : SOCIALS
+    return (
+        <div className="owid-socials">
+            <p className="owid-socials__heading">Follow us</p>
+            <ul className="owid-socials__list">
+                {items.map(({ title, url, icon }) => (
+                    <li className="owid-socials__item" key={title}>
+                        <a
+                            className="owid-socials__link"
+                            href={url}
+                            title={title}
+                            target="_blank"
+                            rel="noopener"
+                            data-track-note="latest_page_follow_us"
```

Comment:
should the track note be more context agnostic (or take the context into account)? This might be a bit misleading when looking at GA events